snippets/objectscript-class.json

```diff
@@ -70,7 +70,7 @@
     "description": "Relationship"
   },
   "BusinessService": {
-		"prefix": ["BusinessService","Interoperability"],
+		"prefix": ["BusinessService","Interoperability","ClassOperation"],
```

Comment:
Did you mean `ClassService` here?

Reply:
Thanks for this review, you are right, PR updated